torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -622,7 +622,8 @@ at::Tensor AtenXlaType::mm(const at::Tensor& self,
 at::Tensor AtenXlaType::matmul(const at::Tensor& self,
                                const at::Tensor& other) const {
   return bridge::AtenFromXlaTensor(XLATensor::matmul(
-      bridge::GetXlaTensor(self), bridge::GetXlaTensor(other)));
+      bridge::GetXlaTensor(self), bridge::GetXlaTensor(other),
+      /*use_full_conv_precision=*/s_use_full_conv_precision_));
```

Comment:
No please, let's get rid of this.
It is a global flag anyway, and there is no reason to poison all the APIs from up the call chain.
Let's have a global (like currently have) flag, maybe initialized by environment as well, and leave the APIs alone.
Only when we call the lower level Dot/Conv/DotGeneral/... we will ask for the global flag value.
Stick it in helpers.cpp/h

Reply:
Done.